rust/datafusion/src/sql/planner.rs

```diff
@@ -343,7 +346,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
     }
 
     /// Generate a logic plan from an SQL select
-    fn select_to_plan(&self, select: &Select) -> Result<LogicalPlan> {
+    pub fn select_to_plan(&self, select: &Select) -> Result<LogicalPlan> {
```

Comment:
Does this need to be made `pub`?

Reply:
Oh, this has nothing to do with the current PR, I'm using this function in my project, so I added pub